tsconfig.json

```diff
@@ -1,18 +1,15 @@
 {
   "compilerOptions": {
-    "declaration": true,
     "esModuleInterop": true,
+    "forceConsistentCasingInFileNames": true,
+    "lib": ["ES2020"],
     "module": "CommonJS",
     "moduleResolution": "node",
-    "outDir": "dist",
-    "sourceMap": true,
+    "noEmit": true,
+    "skipLibCheck": true,
     "strict": true,
-    "target": "ES2017",
-    "typeRoots": [
-      "./node_modules/@types",
-    ],
+    "target": "es2017",
+    "typeRoots": ["./node_modules/@types"]
```

Comment:
We shouldn't need this setting, as this is the default value: https://www.typescriptlang.org/tsconfig#typeRoots

Reply:
Well, take that back, this tells TypeScript to use `@types/*` packages that are located immediately within `node_modules` as opposed to anywhere else, but we don't usually have to mess with this setting.

Reply:
Yeah, good point. I just preserved it because it was already there. Removed in [042dd7a](https://github.com/MetaMask/json-rpc-middleware-stream/pull/25/commits/042dd7a4ef1e32ccd5ee437b50d90330062b877a)